scripts/nginx/moviematch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat > /etc/nginx/apps/moviematch.conf << NGINX
+    location ^~ /moviematch/ {
+        proxy_pass http://127.0.0.1:8420/;
+        proxy_set_header Upgrade \$http_upgrade;
+    }
+NGINX
+
+echo "host: 0.0.0.0
+basePath: /moviematch" >> /opt/moviematch/config.yaml
```

Comment:
`basePath` is renamed back to `rootPath` in the next release since I was getting too many bugs with people confused about the inconsistency between v1 and v2. Something to be aware of!